src/main/java/data/SheetsConverter.java

```diff
@@ -33,18 +28,20 @@
     "1QnVlh-pZHycxzgQuk0MN2nWOY6AGu9j4wGZaGzi_W9A";
   private final String ACCOUNT_RANGE = "'sheet2'!A1";
   private final String VENDOR_RANGE = "'sheet1'!A1";
+  private final String TAB_1 = "Sheet1";
+  private final String TAB_2 = "Sheet2";
   private final String INPUT_OPTION = "RAW";
 
   public boolean updateSheets(ArrayList<Vendor> vendorList, String accessToken) 
     throws GeneralSecurityException, IOException {
-    writeToSheet(vendorList, accessToken); 
+    sheetsService = SheetsServiceUtil.getSheetsService(accessToken);
+    clearSheets(sheetsService);
+    writeToSheet(vendorList, sheetsService); 
     return true;
   }
```

Comment:
Could you explain how errors are handled/going to be handled? I don't know a ton about the sheets API, but I think it make it easier to use/debug these methods

Reply:
I think just having the error be thrown would work best. I think returning a boolean isn't the best solution now because I don't catch the exceptions if they happen now. When you call the update sheets method, you should throw a try/catch around the call and handle any exceptions that might be thrown(GeneralSecurityException, IOException).